homework.md

```diff
@@ -4,77 +4,78 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
Hi Shadi, I think the question doesn't need to be changed, only the answer.

Reply:
Hiya, thanks for your comment. I didn`t notice that the question changed, as I only checked VS